marketing_transactional.go

```diff
@@ -0,0 +1,52 @@
+package hubspot
+
+import "fmt"
+
+const transactionalBasePath = "transactional"
+
+// DealService is an interface of deal endpoints of the HubSpot API.
```

Comment:
Comment here and one for TransactionalServiceOp seem to be copied from DealService. Could you make these comments fit for Transactional Service?
As you may know, GoDoc starts from the target resource name (i.e. `TransactionalService` in this case) appreciate if you can follow this practice!

Reply:
Comments fixed.